### DISCUSSION TITLE
Missing project references when starting from the Uno Platform App template. (Working with Wasm Head)

### DISCUSSION THREAD

**Discussion by poepoe12002** [unoplatform/uno.extensions#1335] [General]
Steps to reproduce the problem - Tried this several times over two days.  
**It did work once, so maybe this is a me problem.**  

Fire up VS
Select New Project
Select 'Uno Platform App' template
Choices **(FYI - Don't seem to matter)**
	Default set of options (Customized)
	Framework:	.NET 6.0
	Platforms:	WebAssembly **Only**
	Presentation:	MVVM
	Projects:		Not Server
	Testing:		Not Unit Tests | Not UI Tests - **FYI - They are added anyway**
	Features:		Not PWA Manifest | Not VS Code Debugging
	Extensions:	DI, Configuration, Http, Localization, Navigation - Regions, Logging - Default
	Application:	No Change
	Theme:		Material
Select Create

Build
Multiple Errors of Type: CS0234	The type or namespace name 'DataContracts' does not exist in the namespace 'TrashWasm003' (are you missing an assembly reference?)
Resolution: Added project reference 'TrashWasm003.DataContracts' to 'TrashWasm003'

Build
Multiple Errors of Type: CS0246	The type or namespace name 'App' could not be found (are you missing a using directive or an assembly reference?)
Resolution: Added project reference 'TrashWasm003' to 'TrashWasm003.Wasm'

Successful Build

**Comment by jeromelaban** [comment]
Thanks for the report, let's track in https://github.com/unoplatform/uno.extensions/issues/1337

**Comment by nickrandolph** [comment]
As a follow up we've been unable to replicate the build errors. However, we have fixed the issue where test projects are created when none are selected - this will come in the next release of the wizard.

@poepoe12002 if you're able to reproduce the build errors, can you post another comment here and we'll reopen this issue.
